docs/common/agent-builder/_saving-your-tree.mdx

```diff
@@ -0,0 +1,3 @@
+Save your tree early and often! Any changes made to your AI will be lost if you leave the Agent Builder without saving.
```

Comment:
This is irrelevant to this PR, but I think it would be nice to use the History API to add a little "Are you sure?" modal before navigating back, or closing the tab, with your browser.

Reply:
Agree, this is something [we've logged](https://linear.app/regression-games/issue/REG-1038/bt-gpt-agent-builder-do-not-allow-navigation-with-unsaved-changes) but haven't prioritized